packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -516,4 +516,60 @@ describe('masternode', () => {
       expect(data.length).toBe(64)
     })
   })
+
+  describe('accountHistoryCount', () => {
+    it('should get accountHistoryCount', async () => {
+      await waitForExpect(async () => {
+        const accountHistoryCount = await client.account.accountHistoryCount()
```

Comment:
The variable name accountHistoryCount should be count like below

Reply:
Changed @jingyi2811